lib/tyxml/tyxml_js.ml

```diff
@@ -42,13 +42,10 @@ module Xml = struct
   type keyboard_event_handler = Dom_html.keyboardEvent Js.t -> bool
   type attrib_k =
     | Event of biggest_event_handler
-    | Attr of Dom.attr Js.t
+    | Attr of Js.js_string Js.t option React.S.t
```

Comment:
Would it make sense (from an efficiency perspective) to keep a constant attribute constructor, with the new signal attribute ?